.config/samples-config-v1.json

```diff
@@ -0,0 +1,175 @@
+            "Word",
+            "Excel",
+            "PowerPoint"
+        ],
+        "languages": [
+            "TS",
+            "JS"
+        ],
+        "technologies": [
+            "Custom Function",
+            "SSO",
+            "Graph"
+        ]
+    },
+    "samples": [
+        {
+            "id": "Excel-HelloWorld-TaskPane-JS",
+            "onboardDate": "2024-07-01",
+            "title": "Excel Hello World Task Pane Add-in",
+            "shortId": "Excel Hello World",
+            "shortDescription": "A simple Excel Task Pane Add-in using JavaScript.",
+            "fullDescription": "This sample demonstrates how to create a simple Excel Task Pane Add-in using JavaScript.",
+            "types": [
+                "Excel"
+            ],
+            "tags": [
+                "JS",
+                "Hello World",
+                "Excel"
+            ],
+            "thumbnailPath": "assets/thumbnail.png",
+            "gifPath": "assets/sampleDemo.gif",
+            "suggested": false
+        },
+        {
+            "id": "Word-HelloWorld-TaskPane-JS",
+            "onboardDate": "2024-07-01",
+            "title": "Word Hello World Task Pane Add-in",
+            "shortId": "Word Hello World",
+            "shortDescription": "A simple Word Task Pane Add-in using JavaScript.",
+            "fullDescription": "This sample demonstrates how to create a simple Word Task Pane Add-in using JavaScript.",
+            "types": [
+                "Word"
+            ],
+            "tags": [
+                "JS",
+                "Hello World",
+                "Word"
+            ],
+            "thumbnailPath": "assets/thumbnail.png",
+            "gifPath": "assets/sampleDemo.gif",
+            "suggested": false
+        },
+        {
+            "id": "Word-Add-in-AIGC",
+            "onboardDate": "2024-03-15",
+            "title": "Create a Word AIGC Add-in",
+            "shortId": "Word AIGC",
+            "shortDescription": "Inserting content generated by AI or predefined into document.",
+            "fullDescription": "This Add-in shows capabilities to insert content generated by AI or predefined into document and customize the style of document.",
+            "types": [
+                "Word"
+            ],
+            "tags": [
+                "TS",
+                "AIGC",
+                "Word"
+            ],
+            "thumbnailPath": "assets/thumbnail.png",
+            "gifPath": "assets/sampleDemo.gif",
+            "suggested": false
+        },
+        {
+            "id": "Word-Add-in-WritingAssistant",
+            "onboardDate": "2024-03-06",
+            "title": "Word Writing Assistant",
+            "shortId": "Word Writing Assistant",
+            "shortDescription": "A Word Add-in to assist writing with word document.",
+            "fullDescription": "This add-in demonstrates Word add-in capabilities to help users check errors, improve writing, and rephrase content.",
```

Comment:
Is this sample showing how the inherent capabilities of Word add-ins (i.e., Office.js) can help? Or that Word add-ins let you connect to third-party libraries that do this?

Reply:
I would guess the writing assistant is one of the major scenarios of Word JS APIs so that a showcase of how to use these APIs might help. I think this question should better be answered by our PM @MingjiaLiu1995 .

Reply:
I remember this sample is to showcase the Annotation APIs and the canvas card, including add annotation, ignore annotation, and show canvas card etc. Also add @wangyun-microsoft for notice.

Reply:
This sounds like a collection of snippets that is showing off APIs, rather than a customer-scenario. I would vote to not include it. We have Script Lab to show off individual APIs.

Reply:
@MingjiaLiu1995 @AlexJerabek  I removed Writing assistant for now. I'll add it again once you make an agreement.

Reply:
First off, if your document is too short, the "Step 2" button doesn't do anything. Without some guidance, users might assume the sample is broken when nothing happens.

Second, my original point about the description is that it makes it sound like Word APIs help with writing assistance. However, it seems more than we provide a couple APIs (annotations) that help writing assistants implement their add-ins. I don't find it very clear in the sample code where I'd replace parts with my own assistance model. Seems like too much is hard-coded at the moment.

Third, I find the use of arbitrary selections and text is confusing. I would be less confused with a sample piece of text with clear issues that could be analyzed. Like, if the sample searched for the word "very" and suggested that word be removed. Or some common programming acronym was replaced with the spell-out version.